frontend/src/core/api/machinery.js

```diff
@@ -42,9 +42,8 @@ export default class MachineryAPI extends APIBase {
         return results.map(item => {
             return {
                 sources: [{
-                    type: 'Translation memory',
+                    type: 'translation-memory',
                     url: '/',
```

Comment:
You can move the `url`s too.

Reply:
only the `url` which have fixed string or all of them? Cause I thought about including the url in the react component too. But there are some like for example `MicrosoftTerminology` which has `item.source` and `locale.msTerminologyCode` as its `url`

Reply:
I think all of them. For the ones that need some data, you can pass the data down to the component. That makes for a more consistent approach.